src/server/CollisionDetector.cpp

```diff
@@ -1,24 +1,35 @@
 #include "CollisionDetector.hpp"
 #include <iostream>
-#include "../common/util.hpp"
-#include "CollisionHandler.hpp"
 
 CollisionDetector::CollisionDetector() {
     //
 }
 
-// TODO: Move collision handling out and make groups param const
-void CollisionDetector::detectGroupCollisions(std::vector<Group*> groups) {
-    for(Group* group_a: groups) {
-        for(Group* group_b: groups) {
+void CollisionDetector::handleGroupShapeCollision(GroupShape* group_a, GroupShape* group_b) {
```

Comment:
this logic could use a comment or documentation

Reply:
Done in 9d5a236